xknx/io/request_response.py

```diff
@@ -58,14 +58,10 @@ def response_rec_callback(self, knxipframe, _):
 
     def on_success_hook(self, knxipframe):
         """Do something after having received a valid answer. May be overwritten in derived class."""
-        self.xknx.logger.debug(
-            "Success: Received correct answer from KNX bus: %s",
-            knxipframe.body.status_code,
-        )
 
     def on_error_hook(self, knxipframe):
         """Do something after having received error within given time. May be overwritten in derived class."""
-        self.xknx.logger.warning(
+        self.xknx.logger.debug(
```

Comment:
I think this is a noteworthy error message. Why do we have to log this only in debug?

Reply:
HA doesn't allow it since this only happens on connection errors. Users will have a warning once disconnected and an info once reconnected. As a user, this would suffice my needs - not only for HA. If they need more info they can still turn on debug logging.